codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2FrameWriter.java

```diff
@@ -130,7 +133,15 @@ public int maxFrameSize() {
     }
 
     @Override
-    public void close() { }
+    public void close() {
+        if (headersEncoder instanceof AutoCloseable) {
+            try {
+                ((AutoCloseable) headersEncoder).close();
```

Comment:
Ok, my bad. 

I thought Java 7 is a baseline, I will replace `AutoCloseable` -> `Closeable`